browser/src/control/Control.TopToolbar.js

```diff
@@ -177,6 +177,8 @@ class TopToolbar extends JSDialog.Toolbar {
 			{type: 'toolitem',  id: 'numberformatdecdecimals', text: _UNO('.uno:NumberFormatDecDecimals', 'spreadsheet', true), visible: false, command: '.uno:NumberFormatDecDecimals'},
 			{type: 'toolitem',  id: 'numberformatincdecimals', text: _UNO('.uno:NumberFormatIncDecimals', 'spreadsheet', true), visible: false, command: '.uno:NumberFormatIncDecimals'},
 			{type: 'separator', orientation: 'vertical',   id: 'break-number', visible: false},
+			{type: 'button', id: 'gridvisible', img: 'gridvisible', hint: _UNO('.uno:GridVisible'), uno: 'GridVisible', hidden: true},
+			{type: 'button', id: 'griduse', img: 'griduse', hint: _UNO('.uno:GridUse'), uno: 'GridUse', hidden: true},
```

Comment:
`uno:` ? shoudn be `command: .uno:XXX` I think

Reply:
Hmm, it works. I guess ".uno" is added inside "_extractCommand". Didn't check but since it works, that code should be adding the prefix :)